roles/glusterfs/handlers/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Restart mesos services
+  service: name={{item}} state=restarted
+  with_items:
```

Comment:
But from my understanding mesos no longer writes to `/mnt`, or am I mistaken?
If I understand correctly we want `/mnt` to only be used by applications. In this case restarting Mesos and Marathon will not restart all applications since docker images would continue to run. Maybe if you also restart docker ;)